### DISCUSSION TITLE
Change license of GeometryService?

### DISCUSSION THREAD

**Discussion by dkowollik** [ansys/pyansys-geometry#1390] [Q&A]
Hi @RobPasMue,

is it possible to change the license preference of the GeometryService to the alternative Mechanical Enterprise Prep Post license (increment: preppost) like in Discovery and SpaceClaim?

**Comment by RobPasMue** [comment]
Hi @dkowollik! I think this is related to https://github.com/ansys/pyansys-geometry/discussions/754 and the corresponding answer https://github.com/ansys/pyansys-geometry/discussions/754#discussioncomment-7082912

The Ansys Geometry Service is required to use a Discovery Modeling license... However, you can use PyAnsys Geometry with SpaceClaim or Discovery, which allow the usage of other license types in that case. Of course, performance might be slower in those cases because the final purpose of the Geometry Service is to provide a fast, headless CAD service.

**Reply by dkowollik** [reply]
Hi @RobPasMue! Do you plan to add this license option to the GeometryService? This makes no sense compared to all other Anys license strategies with Mechanical Enterprise Prep Post.

**Reply by RobPasMue** [reply]
Hi @dkowollik! Let me ask around where Ansys stands about this point and get back to you :)
